src/app/pages/task-calendar/cron/cron-form/cron-form.component.ts

```diff
@@ -74,8 +75,8 @@ export class CronFormComponent {
            type: 'scheduler',
            name: 'cron_picker',
            placeholder: T('Schedule a Cron Job'),
-           tooltip: T('Choose one of the convenient presets\
-             or choose <b>Custom</b> to trigger the advanced scheduler UI'),
+	   tooltip: T('Select one of the convenient presets or <i>Custom</i>\
+	   to open the advanced cron job scheduler UI.'),
```

Comment:
Please don't say "UI".  Most users don't know what that is:

to open the advanced scheduler.